service/controller/chart/resource/release/create.go

```diff
@@ -82,8 +83,12 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 	// If we do timeout the install will continue in the background.
 	// We will check the progress in the next reconciliation loop.
 	go func() {
+		if skipCRDs {
+			r.logger.Debugf(ctx, "helm release %#q has SkipCRDs set to true, not installing release CRDs", releaseState.Name)
```

Comment:
```suggestion
			r.logger.Debugf(ctx, "helm release %#q has SkipCRDs set to true, not installing CRDs", releaseState.Name)
```

I think we can drop the 2nd release. WDYT?